.travis.yml

```diff
@@ -13,4 +13,4 @@ install:
   - travis_retry composer install
 
 script:
-  - phpunit
\ No newline at end of file
+  - phpunit -c phpunit.xml --tap --coverage-text
```

Comment:
Sorry, I did not mean commits. I meant "test run". If we test of 5 different PHP versions, why should we run the coverage on all of them? The coverage will be the same, right? It takes simnifically more time to run the tests if you want to calculate the coverage. That is why you only run the coverage once per CI run.